tsconfig.json

```diff
@@ -23,5 +23,9 @@
     "typeRoots": ["node_modules/@types", "src/types"]
   },
   "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
-  "exclude": ["node_modules"]
+  "exclude": [
+    "node_modules",
+    "/src/components/ConfirmPopover/index.tsx",
+    "src/components/ConfirmPopover/index.tsx"
```

Comment:
If we are not doing the Chakra update now, I suggest to add a comment to remove those lines.

Reply:
this is a mistake